tests/src/Simple/HelloWasm/Program.cs

```diff
@@ -25,6 +25,13 @@ private static unsafe void Main(string[] args)
 			PrintString(s, 14);
 		}
 		
+		var boxedInt = tempInt;
+		if(((int)boxedInt) == 9)
```

Comment:
This doesn't actually box. You'd need something like:
object boxedInt = (object)tempInt;

Reply:
when I split this PR out from the ldfld one I just did a bad job with copy paste, thanks for spotting it